File: app/src/main/java/com/niel/circularprogresscustomview/ProgressPieActivity.java
package com.niel.circularprogresscustomview;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.SeekBar;

public class ProgressPieActivity extends AppCompatActivity {


  private SeekBar mSeekBar;

  private PieProgressView mProgressPieView;
  @Override protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_progress_pie);


    mProgressPieView = (PieProgressView) findViewById(R.id.progressPie);

    // SeekBar
    mSeekBar = (SeekBar) findViewById(R.id.seekBar);
    mSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
      @Override
      public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
        mProgressPieView.setProgress(i);
      }

      @Override
      public void onStartTrackingTouch(SeekBar seekBar) {

      }

      @Override
      public void onStopTrackingTouch(SeekBar seekBar) {

      }
    });
  }
}

File: app/src/main/java/com/niel/circularprogresscustomview/ProgressPieActivity.kt
package com.niel.circularprogresscustomview

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.SeekBar

class ProgressPieActivity : AppCompatActivity() {


    private var mSeekBar: SeekBar? = null

    private var mProgressPieView: PieProgressView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_progress_pie)


        mProgressPieView = findViewById(R.id.progressPie)

        // SeekBar
        mSeekBar = findViewById(R.id.seekBar)
        mSeekBar!!.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
            override fun onProgressChanged(seekBar: SeekBar, i: Int, b: Boolean) {
                mProgressPieView!!.setProgress(i)
            }

            override fun onStartTrackingTouch(seekBar: SeekBar) {

            }

            override fun onStopTrackingTouch(seekBar: SeekBar) {

            }
        })
    }
}
